### ISSUE TITLE
[0.2] create database statement fails, going to wrong endpoint

### ISSUE THREAD

**Issue by beckettsean** [influxdb/telegraf#367]
Installed telegraf 0.2.0 from deb package. On launch, it's trying to create the `telegraf` database in the InfluxDB instance, but it's sending the command to the wrong endpoint: 

```
[http] 2015/11/13 13:07:27 127.0.0.1 - - [13/Nov/2015:13:07:27 -0500] POST /write?consistency=&db=telegraf&precision=s&q=CREATE+DATABASE+telegraf&rp= HTTP/1.1 404 55 - InfluxDBClient 661c5c14-8a31-11e5-8012-000000000000 1.687633ms
```

That request should be a `GET` to the `/query` endpoint, not a `POST` to the `/write` endpoint.

**Comment by beckettsean** [comment]
Once I manually created the `telegraf` database everything works as expected, but this is a poor "out-of-the-box" experience.

**Comment by sparrc** [comment]
@beckettsean This will be fixed in 0.2.1, it was actually a bug in the InfluxDB v2 client, see https://github.com/influxdb/influxdb/pull/4700 and #351

**Comment by sparrc** [comment]
0.2.2 is now released and has this fix
